library/ReplicatedTable.cs

```diff
@@ -186,8 +183,9 @@ public bool DeleteIfExists()
                 // Create individual table replicas if they are not created already 
                 foreach (var entry in txnView.Chain)
                 {
-                    CloudTable ctable = entry.Item2.GetTableReference(this.TableName);
-                    if (ctable.DeleteIfExists())
+                    TableClient ctable = entry.Item2.GetTableClient(this.TableName);
+                    var resp = ctable.Delete();
+                    if (resp.Status != (int)HttpStatusCode.NotFound)
```

Comment:
should we looks for a specific status for success instead of relying on !NotFound ? What if there is some other error on storage side for delete ?

Reply:
Good catch, will change this to ` if (resp.Status == (int)HttpStatusCode.NoContent)`
![image](https://github.com/Azure/rtable/assets/122410443/8072e9e1-e45c-40ca-bf2d-fd469ea3f970)